root/etc/s6-overlay/s6-rc.d/init-transmission-config/run

```
@@ -0,0 +1,49 @@
+if [[ ! -f /config/settings.json ]]; then
+    cp /defaults/settings.json /config/settings.json
+fi
+
+if [[ -n "$USER" ]] && [[ -n "$PASS" ]]; then
+    sed -i '/rpc-authentication-required/c\    "rpc-authentication-required": true,' /config/settings.json
+    sed -i "/rpc-username/c\    \"rpc-username\": \"$USER\"," /config/settings.json
+    sed -i "/rpc-password/c\    \"rpc-password\": \"$PASS\"," /config/settings.json
+else
+    sed -i '/rpc-authentication-required/c\    "rpc-authentication-required": false,' /config/settings.json
+    sed -i "/rpc-username/c\    \"rpc-username\": \"$USER\"," /config/settings.json
+    sed -i "/rpc-password/c\    \"rpc-password\": \"$PASS\"," /config/settings.json
+fi
+
+if [[ -n "$WHITELIST" ]]; then
+    sed -i '/rpc-whitelist-enabled/c\    "rpc-whitelist-enabled": true,' /config/settings.json
+    sed -i "/\"rpc-whitelist\"/c\    \"rpc-whitelist\": \"$WHITELIST\"," /config/settings.json
+else
+    sed -i '/rpc-whitelist-enabled/c\    "rpc-whitelist-enabled": false,' /config/settings.json
+    sed -i "/\"rpc-whitelist\"/c\    \"rpc-whitelist\": \"$WHITELIST\"," /config/settings.json
+fi
+
+if [[ -n "$HOST_WHITELIST" ]]; then
+    sed -i '/rpc-host-whitelist-enabled/c\    "rpc-host-whitelist-enabled": true,' /config/settings.json
+    sed -i "/\"rpc-host-whitelist\"/c\    \"rpc-host-whitelist\": \"$HOST_WHITELIST\"," /config/settings.json
+else
+    sed -i '/rpc-host-whitelist-enabled/c\    "rpc-host-whitelist-enabled": false,' /config/settings.json
+    sed -i "/\"rpc-host-whitelist\"/c\    \"rpc-host-whitelist\": \"$HOST_WHITELIST\"," /config/settings.json
+fi
+
+if [[ -n "${PEERPORT}" ]]; then
+    sed -i "/\"peer-port\"/c\    \"peer-port\": ${PEERPORT}," /config/settings.json
+    sed -i '/peer-port-random-on-start/c\     "peer-port-random-on-start": false,' /config/settings.json
+fi
+
+# permissions
+chown abc:abc \
+    /config/settings.json \
+    /downloads \
+    /downloads/complete \
+    /downloads/incomplete \
```

Why this change is required? For nfs mounted volumes it causes init issues on my end:
```
chown: changing ownership of '/downloads/complete': Operation not permitted
chown: changing ownership of '/downloads/incomplete': Operation not permitted
```

---

It is not a change, this file has only been moved. It would have had this error before this rebase too, but the default behaviour was to ignore errors.